tests/components/side-panel/toolbar.spec.tsx

```diff
@@ -0,0 +1,56 @@
+import { fireEvent, render, screen } from "@testing-library/react";
+import "jest-canvas-mock";
+import { Toolbar } from "src/components/side-panel/toolbar";
+import { TestHelper } from "src/util/test-helper";
+
+describe("toolbar", () => {
+    afterEach(() => {
+        jest.restoreAllMocks();
+        jest.resetAllMocks();
+    });
+
+    it("should collapse toggle panel if it is expanded on clicking the mode button", () => {
+        renderComponent();
+        fireEvent.click(getIconButton());
+        expect(mockToggleMode).toHaveBeenCalled();
+    });
```

Comment:
remove this test as we are not collapsing the panel on clicking any of the mode buttons